zutil_p.h

```diff
@@ -0,0 +1,34 @@
+/* zutil_p.h -- Private inline functions used internally in zlib-ng
+ *
+ */
+
+#ifndef ZUTIL_P_H
+#define ZUTIL_P_H
+
+#ifdef __APPLE__
+#  include <stdlib.h>
+#else
+#  include <malloc.h>
+#endif
+
+/* Function to allocate 16 or 64-byte aligned memory */
+static inline void *zng_alloc(unsigned size) {
```

Comment:
Parameter type should be size_t, because it avoids zeroing the upper bits on 64-bit machines.

Reply:
Fixed